Testscripts/Linux/verify_vmbus_heartbeat_properties.sh

```diff
@@ -19,6 +19,14 @@
 # Source constants file and initialize most common variables
 UtilsInit
 
+# Install python 2.7 if missing
+python_path=$(which python)
+LogMsg "Checked the system Python path: $python_path"
+if [[ $? -gt 0 ]]; then
```

Comment:
$? return last execution command, here it will return value for LogMsg.

Reply:
Thanks. Fixed in 4391bf2aca271a78a861743d19b71bb2fe43532e